include/uxr/agent/utils/SeqNum.hpp

```diff
@@ -30,16 +30,16 @@ class SeqNum
     /*
      * Operators.
      */
-    inline operator uint16_t() const { return seq_num_; }
+    operator uint16_t() const { return seq_num_; }
     operator int() const { return seq_num_; }
 
-    inline SeqNum& operator+=(const int& rhs)
+    SeqNum& operator+=(const int& rhs)
```

Comment:
Because it allows operating with `int` directly, for examples:
```
SeqNum a = 0xFFFF;
a += 1;
```
In other case you shall do this:
```
SeqNum a = 0xFFFF;
a += uint16_t(1);
```